### PR TITLE
FDA-165 Build Tobacco Adverse Events Frontend.

### PR DESCRIPTION
**Pull request by nagkuchipudi**
FDA-165 Build Tobacco Adverse Events Frontend.

### CODE REVIEW

File: src/pages/apis/tobacco/event/_fields.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    possible_values:
+    type: string
+  reported_health_problems:
+    description: "Text reflecting the MedDRA terms selected by the reporter.  When two or more terms were reported, they are all displayed and separated by forward slashes. If there are duplicate terms reported such as cough/dry eye, both are displayed. When no health problem is reported the text “No information provided” is displayed."
+    format:
+    is_exact: true
+    possible_values:
+    type: string
+  reported_product_problems:
+    description: "Text reflecting the SRP categorical list of values. When two or more values were reported, they are all displayed and separated by forward slashes. When no product problem is reported the text “No information provided” is displayed."
```
Comment (beardedfinch) [inline src/pages/apis/tobacco/event/_fields.yaml:51]: Let's take out the line about "When two or more terms" and reformat this to represent it as an array of strings.

File: src/pages/apis/tobacco/event/_explorers.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+oneReport:
```
Comment (beardedfinch) [inline src/pages/apis/tobacco/event/_explorers.yaml:1]: Let's change the name & title to something like "Range of Submission Dates".

File: src/constants/fields/tobacco.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    is_exact: false
+    possible_values:
+    type: string
+  nonuser_affected:
+    description: "The reported age of the consumer at the time of the adverse event report, expressed in the unit in the field `age_unit`."
+    format:
+    is_exact: false
+    possible_values:
+    type: string
+  meta:
```
Comment (beardedfinch) [inline src/constants/fields/tobacco.yaml:56]: Remove `meta` entirely and make the same changes as in the other yaml.

File: src/components/EndpointBox.jsx [hunk 4]
```diff
@@ -121,6 +128,9 @@ const EndpointBox = (props: tPROPS) => {
     'other': {
       'nsde': <div className="ep-icon" style={bg_color['other']}><i className="fa fa-3x fa-database" style={{color: "white"}}/></div>,
       'substance': <div className="ep-icon" style={bg_color['other']}><i className="fa fa-3x fa-flask" style={{color: "white"}}/></div>
+    },
+    'tobacco': {
+      'event': <div className="ep-icon" style={bg_color['tobacco']}><i className="fa fa-3x fa-database" style={{color: "white"}}/></div>
```
Comment (beardedfinch) [inline src/components/EndpointBox.jsx:133]: fa-leaf should be a more applicable one than fa-database.

File: src/components/Nav.jsx [hunk 5]
```diff
@@ -162,6 +162,7 @@ const Nav = (props: tPROPS) => {
                     <Link className={linkCx} to='/apis/device/' onClick={closeMobileNav}>Device Endpoints</Link>
                     <Link className={linkCx} to='/apis/food/' onClick={closeMobileNav}>Food Endpoints</Link>
                     <Link className={linkCx} to='/apis/other/' onClick={closeMobileNav}>Other Endpoints</Link>
+                    <Link className={linkCx} to='/apis/tobacco/' onClick={closeMobileNav}>Tobacco Endpoints</Link>
```
[the remark this answers is not in the document]
Comment (nagkuchipudi) [reply src/components/Nav.jsx:165]: These files are missed in the PR, I'll update them.

**Pull request merged**